device-types/QNAP/TS-EC1280U.yaml

```diff
@@ -0,0 +1,43 @@
+---
+manufacturer: QNAP
+model: TS-EC1280U
+slug: qnap-ts-ec1280u
+part_number: TS-EC1280U
+u_height: 2
+airflow: front-to-rear
+is_full_depth: true
+weight: 14.18
+weight_unit: kg
+subdevice_role: parent
```

Comment:
This is generally used for chassis which house other devices, this is not meant for devices with modules. Is this actually needed, and if so why are there no `device-bays`?